Patches/MeetingHudPatch.cs

```diff
@@ -1210,24 +1207,16 @@ public static void Postfix(MeetingHud __instance)
 
                     } */
             //インポスター表示
+            sb.Append(seerRoleClass?.GetMark(seer, target));
```

Comment:
Can you sync with the latest commit, since I already added `GetMark` support for modded players